sidebars.js

```diff
@@ -1225,7 +1230,7 @@ const celoL2Sidebar = [
       },
       {
         type: "doc",
-        label: "Pectra Upgrade",
+        label: "L1 Pectra Upgrade",
```

Comment:
Could you highlight here that for Mainnet, running the migration version is enough? It's in the callout, but not the first thing that catches a reader's attention; I'd add a title for Mainnet.